_posts/2016-12-14-four-tips-sass-variables.markdown

```diff
@@ -0,0 +1,237 @@
+---
+layout: post
+title: "Four Tips for Using Sass Variables"
+date: 2016-12-14
+categories: front-end sass css
+author: Max Antonucci
+excerpt: Sass variables help keep your CSS efficient and flexible, but they're more complex than you may think. These are four basic tips to get the most out from your Sass variables.
+
+source: internal
+ext-url: none
+---
+
+While [Sass](http://sass-lang.com/) brings many great functions to CSS, one of the biggest is variables. Variables mean no more `ctrl-F`-ing our way through stylesheets when we change a color or margin value. Putting these frequent values in one place makes managing them that much easier.
+
+Turns out there's much more to Sass variables then that basic use. At least if you want to get as much out of them as possible.
+
+These are four basic tips to get a lot more out of Sass variables. They'll help developers use more variables with fewer issues and write DRYer code too. If you're relatively new to Sass and want to expand your understanding, this article's for you.
```

Comment:
It might be helpful to add the Tip number to the headers:
* Tip 1: Know the Different Variable Types (Basic, List, and Map)
* Tip 2: Know the Different Variables Scopes (Global, Single File, and Selector)
* Tip 3: Use a Naming Convention
* Tip 4: Combine Variables and Loops to Save Time